ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+###########
+# OpenSSL #
+###########
+- name: Check if OpenSSL 32bit installed
+  win_stat:
+    path: 'C:\openjdk\OpenSSL-1.1.1-x86_32'
+  register: openssl32_installed
+  tags: openssl
+
+- name: Check if OpenSSL 64bit installed
```

Comment:
You could combine this check into one statement (possibly).  On linux this would work,
` shell: ls -d /cygdrive/c/openjdk/OpenSSL* >/dev/null 2>&1`

Reply:
I've found a way to check if a directory/file exists with Test-Path in powershell. It outputs either true or false which gets saved to the variable on the register line.